linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_card_drivers/bash/shared.sh

```diff
@@ -7,8 +7,8 @@
 . /usr/share/scap-security-guide/remediation_functions
 populate var_smartcard_drivers
 
-grep -qs "card_drivers =" /etc/opensc*.conf && \
-	sed -i "s/card_drivers =.*/card_drivers = $var_smartcard_drivers;/g" /etc/opensc*.conf
-if ! [ $? -eq 0 ]; then
-	sed -i "s/.*card_drivers =.*/        card_drivers = $var_smartcard_drivers;/g" /etc/opensc*.conf
+OPENSC_TOOL="/usr/bin/opensc-tool"
+
+if [ -f "${OPENSC_TOOL}" ]; then
+    ${OPENSC_TOOL} -S app:default:card_drivers:$var_smartcard_drivers
```

Comment:
Looks good to me. This feature is present in OpenSC for 10+ years so I think we can depend on that in the scripts. Indeed, it looks nicer than the seds.